src/main/webapp/app/entities/core/post/post-update.component.ts

```diff
@@ -152,7 +152,6 @@ export class PostUpdateComponent implements OnInit {
             categories: [this.post.categories], // convert into this.fb.array
             tags: [this.post.tags], // convert into this.fb.array
             degaUsers: [this.post.degaUsers || '', Validators.required], // convert into this.fb.array
-            clientId: [this.post.clientId || ''], // delete once backend is fixed
             publishedDate: [this.post.publishedDate || null], // delete once backend is fixed
             createdDate: [this.post.createdDate || null] // delete once backend is fixed
```

Comment:
delete this and check if it is working, if things are breaking then keep it as it is